cli/tests/fixtures/metrics.py

```diff
@@ -0,0 +1,425 @@
+def agent_metrics_node_fixture():
+    """Agent metrics /node fixture
+
+    :rtype: dict
+    """
+
+    return {
+        "datapoints": [
```

Comment:
👍 doing this helped me catch a bug (there are many disk usage fields, I wasn't searching by tag for `path: /`)